pkg/io/video/convert.go

```diff
@@ -3,8 +3,38 @@ package video
 import (
 	"fmt"
 	"image"
+	"image/color"
 )
 
+// imageToYCbCr converts image to *image.YCbCr
+// Note: conversion can be lossy
+func imageToYCbCr(img image.Image) *image.YCbCr {
```

Comment:
It would be better to receive the destination pointer as an argument to reuse allocated memory, and realloc Y, Cb, Cr slices if the size of the image is changed.

Reply:
Makes senses, I'll update it